app/views/articles/search/_meta.html.erb

```diff
@@ -10,7 +10,7 @@
 <meta property="og:description" content="<%= SiteConfig.community_description %>">
 <meta property="og:site_name" content="<%= community_qualified_name %>" />
 
-<meta name="twitter:site" content="@<%= SiteConfig.social_networks_handle %>">
+<meta name="twitter:site" content="@<%= SiteConfig.social_media_handles["twitter"] %>">
```

Comment:
My tests fail on all the meta tags because it seems SiteConfig.social_media_handles does not exist yet. This may be a bigger problem and I'm busy investigating how to fix the tests and also to make sure this doesn't become a problem when someone first sets up and starts running the app (I'm in the process of testing that use case).
If any oss'es have suggestions, please drop them here.